test/unit/fetch-test.js

```diff
@@ -38,3 +38,44 @@ test('fetch without state', function (t) {
     t.equal(error.name, 'UnauthenticatedError', 'rejects with UnauthenticatedError error')
   })
 })
+
+test('fetch with non-string path', function (t) {
+  simple.mock(internals, 'fetchProperties').resolveWith({})
+
+  fetch({
+    url: 'http://example.com',
+    account: {
+      session: {
+        id: 'abc45678'
+      }
+    }
+  }, false)
+
+  t.deepEqual(internals.fetchProperties.lastCall.arg, {
+    url: 'http://example.com/session/account',
+    bearerToken: 'abc45678',
+    path: false
+  }, 'calls fetchProperties with account url and non-string path')
+  // expected behavior??
```

Comment:
`path` in this context is not part of the URL, but the path to the properties you want to return. If it’s undefined, all properties are returned. If path is set to `address`, then only the address value is returned. `false` makes not much sense here, why did you decide to test with it?

See examples in the README for [account.profile.fetch](https://github.com/hoodiehq/hoodie-client-account#accountprofilefetch) 

Note that this example is wrong – I’ll make a separate issue to get it fixed

```js
account.fetch('fullname').then(function (createdAt) {
  alert('Hey there ' + fullname)
})
```

should be 

```js
account.fetch('fullname').then(function (fullname) {
  alert('Hey there ' + fullname)
})
```